tests/test_parse_review.py

```
@@ -793,6 +793,19 @@ def test_detects_rate_limit_without_explicit_marker(self):
         assert data["verdict"] == "SKIP"
         assert "RATE_LIMIT" in data["summary"]
 
+    def test_detects_timeout_marker(self):
+        timeout_text = """Review Timeout: timeout after 120s
+
+APOLLO (correctness) exceeded the configured timeout.
+"""
+        code, out, err = run_parse(timeout_text, env_extra={"REVIEWER_NAME": "APOLLO"})
```

Ruff (0.14.14)

[warning] Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)